css/interactive-guides/main.css

```diff
@@ -59,9 +59,11 @@
 }
 
 .instruction {
+  font-family: Asap;
+  font-weight: 400;
   text-align: left;
-  font-size: 16px;
-  width: 100%;
+  font-size: 14px;
```

Comment:
I talked to Steven about it and he said we can leave it at 14px since the static guides multi-pane layout uses 14px. I looked back at the redlines and the paragraph text for the sections also uses 14px.